drivers/CoreIMU/source/CoreGyroscope.cpp

```diff
@@ -0,0 +1,15 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreGyroscope.h"
+
+namespace leka {
+
+auto CoreGyroscope::getAngularSpeedXYZ() -> std::tuple<dps_t, dps_t, dps_t>
+{
+	auto data = _core_lsm6dsox.updateData();
+	return std::tuple {data.ui.gy.mdps[0] / 1000, data.ui.gy.mdps[1] / 1000, data.ui.gy.mdps[2] / 1000};
```

Comment:
Same here. 

The division by 1000 must be explicitly explained with a conversion function/lambda or better yet with strong types.

Reply:
Or can the conversion be move to the struct so that we have dps publicly everywhere?